parts/kubernetesmastergenerateetcdcertscript.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
This and probably other k8s files under /parts have been moved to the /parts/k8s folder, seems rebase and resolve is needed here.